ui/StatusQ/src/StatusQ/Controls/StatusBaseButton.qml

```diff
@@ -23,6 +23,7 @@ Button {
     }
 
     property StatusAssetSettings asset: StatusAssetSettings { }
+    property Component statusIconComponent: d.statusIcon
```

Comment:
I wonder why we need an extra icon component here? StatusBaseButton is already an AbstractButton which has a builtin support for icons

Reply:
I see; thanks! Still I'd like @alexandraB99 to confirm as well :)

Reply:
The thing that confuses me here is that we both:
- expose this as a public writable property
- and at the same time initialize it with a private component